common/config/src/raw.rs

```diff
@@ -463,6 +466,23 @@ mod tests {
         assert_eq!(config, new_config);
     }
 
+    #[test]
+    fn test_default_glob() {
```

Comment:
There were two other units tests that caught this change, but they were located in `argv` so not sure if this test was necessary but added it anyway.

Reply:
does the order matter in this vec?